doc/api/console.md

```diff
@@ -273,9 +276,7 @@ prints the result to `stdout`:
 
 ```js
 console.time('100-elements');
-for (let i = 0; i < 100; i++) {
-  ;
-}
+for (let i = 0; i < 100; i++) ;
```

Comment:
Would

```js
for (let i = 0; i < 100; i++) {}
```

work?

Some people may be unfamiliar with the syntax of an empty statement.